libs/designer/src/lib/ui/panel/templatePanel/templatePanel.tsx

```diff
@@ -24,11 +28,17 @@ export const TemplatePanel = ({ onCreateClick }: { onCreateClick: (onSuccessfulC
     dispatch(clearTemplateDetails());
   }, [dispatch]);
   const createWorkflowPanelTabs = useCreateWorkflowPanelTabs({
-    onCreateClick: (onSuccessfulCreation: () => void) => onCreateClick(onSuccessfulCreation),
+    onCreateClick: () => onCreateClick(),
```

Comment:
Just pass the prop onCreateClick as

useCreateWorkflowPanelTabs({ onCreateClick })

We will save on function creation